gapis/api/vulkan/api/command_buffer_control.api

```diff
@@ -421,6 +421,7 @@ cmd VkResult vkEndCommandBuffer(
     vkErrorInvalidCommandBuffer(commandBuffer)
   } else {
     buff := CommandBuffers[commandBuffer]
+    _ = buff.Recording
```

Comment:
Something that might be better (and provide users some use)
if (buff.Recording != RECORDING) {
   vkErrInvalidCommandBufferState()
}
That way we both get the dependency, and there is a clear reason for it.